crates/services/relayer/src/mock_db.rs

```diff
@@ -37,27 +40,42 @@ pub struct MockDb {
 }
 
 impl MockDb {
```

Comment:
Hmm, maybe we want to replace it with `mockall`?
It's up to use, but it seems it would reduce the amount of code for you=)

Reply:
Yeh probably but this is unrelated to this PR